tests/collections/tts/models/test_fastpitch.py

```diff
@@ -38,4 +41,23 @@ def test_inference(pretrained_model, language_specific_text_example):
     model, language_id = pretrained_model
     text = language_specific_text_example[language_id]
     parsed_text = model.parse(text)
-    _ = model.generate_spectrogram(tokens=parsed_text)
+
+    # Multi-Speaker
+    speaker_id = None
+    reference_spec = None
+    reference_spec_lens = None
+
+    if hasattr(model.fastpitch, 'speaker_emb'):
+        speaker_id = 0
```

Comment:
Nitpick: could create a helper function to handle the branching logic easier, like:

```
def _get_inputs(model):
    if hasattr(model.fastpitch, 'speaker_emb'):
        return 0, None, None
    if not hasattr(model.fastpitch, 'speaker_encoder'):
        None, None, None
    ...

```

Reply:
agreed. but it seems this add-ons are only applied to fastpitch models, not all. It looks good to just keep as-is.